examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/templates/MCCommandPayloads.zapt

```diff
@@ -0,0 +1,36 @@
+{{> header excludeZapComment=false}}
+
+#import "MCInteractionModelStructs.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MCAbstractPayload : NSObject <NSCopying>
+@end
+
+{{#zcl_clusters}}
+{{#if (isInConfigList name "MatterCastingClusters")}}
+
+// {{asUpperCamelCase name}} cluster:
+{{#zcl_commands}}
+{{#*inline "completeDecl"}}
+
+@interface MC{{cluster}}Cluster{{command}}Params : {{base}}
+{{#zcl_command_arguments}}
+{{> struct_field_decl cluster=parent.parent.name type=type label=label}};
+{{#*inline "oldNameFieldDecl"}}
+{{> struct_field_decl cluster=parent.parent.name type=type label=commandField}};
+{{/inline}}
+{{/zcl_command_arguments}}
+@end
+{{/inline}}
+{{> completeDecl cluster=(asUpperCamelCase parent.name preserveAcronyms=true)
+                 command=(asUpperCamelCase name preserveAcronyms=true)
+                 includeRenamedProperties=false
```

Comment:
Where is includeRenamedProperties used?

Where includeInitWithResponseValue used?

Reply:
Verified, not used. Removed.